extensions/amp-story-interactive/amp-story-interactive.md

```diff
@@ -342,6 +342,10 @@ The `amp-story-interactive` component elements support [`amp-analytics`](https:/
 </amp-analytics>
 ```
 
+## Accessibility considerations
+
+Currently, this component lacks keyboard and screen reader accessibility.
```

Comment:
cc @mszylkowski please remember to remove this when you introduce keyboard/screen reader capabilities.